stories/topics/proc-aap-aws-application-upgrade.adoc

```diff
@@ -0,0 +1,164 @@
+[id="proc-aap-aws-application-upgrade"]
+
+You can upgrade your existing {AAPonAWS} deployment to the newer version. The upgrade process covers upgrading of Automation Hub and Automation Controller. The upgrade process takes roughly the same amount of time as a {AAPonAWS} deployment install. Follow the steps below to ensure a smooth upgrade process.
+
+[NOTE]
+=====
+{AAPonAWS} supports sequential upgrades. All upgrades should be no more than one minor version behind what you are currently upgrading to. For example, in order to upgrade to {AAPonAWS} to 2.2.20230215, you must be on the first version of {AAPonAWS}.
+=====
+
+[NOTE]
+=====
+The upgrade process does not support
+=====
+
+== Requirements
```

Comment:
.Prerequisites

Reply:
I'm not sure but I think I did the change you expected.